openstack-tripleo-ui-deps.spec

```diff
@@ -1,10 +1,10 @@
 %global sname openstack-tripleo-ui-deps
-%global commit ad6314526b80f656b4807ed0f537f3a089c9b87c
+%global commit 6f160f8c6a0299d3027134224aa3eb3ba103731b
```

Comment:
https://github.com/openstack/tripleo-ui/commit/6f160f8c6a0299d3027134224aa3eb3ba103731b
is merge of https://review.openstack.org/445961 approved by @hguemar